packages/checkbox/src/index.mts

```diff
@@ -120,13 +123,12 @@ export default createPrompt(
 
     const message = chalk.bold(config.message);
 
-    const lines = items
-      .map((item, index) => renderItem({ item, isActive: index === active }))
-      .join('\n');
-
-    const page = usePagination(lines, {
+    const page = usePagination<Item<Value>>({
+      items,
       active,
+      render: renderItem,
```

Comment:
I don't think `render` is a good name here since it renders a single item; so I'd keep the name `renderItem` inside and outside.